### ISSUE TITLE
[css-images-3] Interpolating A with cross-fade(p% A, B)

### ISSUE THREAD

**Issue by LeaVerou** [w3c/csswg-drafts#2853]
Based on [the currently proposed cross-fade() interpolation algorithm](https://github.com/w3c/csswg-drafts/issues/2852#issuecomment-401676199), the titular example would produce something like `cross-fade(x% A, cross-fade(p% A, B))`, x∈[0, 100].

It seems far more sensible to promote the first operand to `cross-fade(100% A, B)` so that it can be interpolated by just the percentage (`cross-fade(x% A, B)`, x∈[p, 100]), which avoids redundant nesting.

**Comment by astearns** [comment]
@LeaVerou should this go on the weekly agenda, since we didn't get to it at the F@F meeting?

**Comment by tabatkins** [comment]
Yeah, we addressed it during the F2F. Just needs edits. Retagging to the correct spec, too.
